diff --git a/Sources/MyHTMLView.h b/Sources/MyHTMLView.h
index e92cc72..33a093a 100644
--- a/Sources/MyHTMLView.h
+++ b/Sources/MyHTMLView.h
@@ -20,7 +20,6 @@
 
 #import <AppKit/AppKit.h>
 #import <WebKit/WebKit.h>
-\
 
 @interface MyHTMLView : WKWebView 
 {
diff --git a/Sources/TSToolbarController.m b/Sources/TSToolbarController.m
index c165d76..533a5e2 100644
--- a/Sources/TSToolbarController.m
+++ b/Sources/TSToolbarController.m
@@ -916,7 +916,7 @@ - (NSToolbarItem *) toolbar: (NSToolbar *)toolbar itemForItemIdentifier: (NSStri
     {
         return [self makeToolbarSymbolsItemWithItemIdentifier:itemIdent key:itemIdent symbolName: @"chevron.right" accessibility: @"Next Page" imageName:@"NextPageAction" newImageName:@"chevron.right" target:self action:@selector(doHtmlNextPage:)];
      }
-\
+
 	if ([itemIdent isEqual: kBackForthKKTID]) {
         
         toolbarItem = [self makeBackForthItemWithItemIdentifier: itemIdent key:itemIdent customView: backforthKK];
