diff --git a/Manual Source/Manual.tex b/Manual Source/Manual.tex
index 297dc49..e66544e 100644
--- a/Manual Source/Manual.tex	
+++ b/Manual Source/Manual.tex	
@@ -3180,7 +3180,7 @@ The Labels item on the toolbar works just like the Tags item, except that it lis
 
 \section{Unicode Panel}
 
-The TeXShop Editor is fully Unicode capable, so  unusual scripts can be entered and typeset. Sometimes it is useful to know the particular Unicode characters being typed, and for that purpose Terada Yusuke provided a Unicode Panel. Select a portion of text, and then select ``Character Info..'' in the Edit Menu. The next page shows this panel examining a piece of Arabic text in a source file to be typeset with XeLaTeX.
+The TeXShop Editor is fully Unicode capable, so unusual scripts can be entered and typeset. Sometimes it is useful to know the particular Unicode characters being typed, and for that purpose Terada Yusuke provided a Unicode Panel. Select a portion of text, and then select ``Character Info..'' in the Edit Menu. Figure \ref{fig:arabic} shows this panel examining a piece of Arabic text in a source file to be typeset with XeLaTeX.
 
 \newpage
 
@@ -3188,7 +3188,7 @@ The TeXShop Editor is fully Unicode capable, so  unusual scripts can be entered
    \centering
    \includegraphics[width=3in]{Graphics/arabic.png} 
    \caption{Arabic Unicode}
-%  \label{fig:example}
+   \label{fig:arabic}
 \end{figure}
 
 \section{Show Invisible}
@@ -3445,23 +3445,23 @@ Unfortunately, the new attribute is totally ignored by the ``Check Spelling'' se
 %
 %
 
-\section{CocoAspell}
+\section{cocoAspell}
 
 
 
-Coco-Aspell is an alternate spell checker by Anton Leuski. Leuski's project allows users to replace Apple's own dictionaries with new dictionaries that can be made ``LaTeX aware'', while still using all the Apple spelling facilities. Leuski made the project open source recently; see \url{https://github.com/leuski/cocoAspell}. 
+cocoAspell is an alternate spell checker by Anton Leuski. Leuski's project allows users to replace Apple's own dictionaries with new dictionaries that can be made ``LaTeX aware'', while still using all the Apple spelling facilities. Leuski made the project open source recently; see \url{https://github.com/leuski/cocoAspell}.
 I think of the project as the ``gold standard'' for TeX-aware spell checking, and if you use it, then the {\em Spell Checking} preference items discussed in the previous section should be turned off.
 
-Coco-Aspell can be difficult to install, and the latest commit to the open source project was August 5, 2017. 
+cocoAspell can be difficult to install, and the latest commit to the open source project was August 5, 2017.
 Leuski understandably stopped working on the project and nobody has stepped up to take it over. Users must then use the dictionaries supplied with cocoAspell, rather than dictionaries by Apple or others.  Below are two documents from 2019, a short description by Koch, and a longer description on installing the code by Schulz.
-These are the latest documents I possess on CocoAspell.
+These are the latest documents I possess on cocoAspell.
 
-\section{Cocoa-Aspell Usage (Koch)}
+\section{cocoAspell Usage (Koch)}
 % Thanks to Anton Leuski for allowing us to distribute cocoAspell in MacTeX. 
 
 cocoAspell is a spell checking service which attaches to the build-in spell checker from Apple. It knows about LaTeX and does not mark LaTeX keywords as misspelled. Since it is not notarized, it cannot be provided in our install packages; obtain it directly from the author's web site.
 
-The spell checker installs a Preference Pane named ``Spelling'' in Apple's System Preference, see Figure \ref{fig:PrefPaneSpellig}. After running the cocoAspell installer, open this pane and configure it appropriately.
+The spell checker installs a Preference Pane named ``Spelling'' in Apple's System Preference, see Figure \ref{fig:PrefPaneSpelling}. After running the cocoAspell installer, open this pane and configure it appropriately.
 
 The Pref Pane requires macOS El Capitan or higher, because Apple changed the requirements for Preference Panes just before El Capitan. The cocoAspell pane meets all current Apple standards.
 
@@ -3472,7 +3472,7 @@ The Pref Pane requires macOS El Capitan or higher, because Apple changed the req
    \centering
    \includegraphics[width=4in]{Graphics/PrefPane.png} 
     \caption{Preference Pane ``Spelling'' in Apple's System Preference}
-    \label{fig:PrefPaneSpellig}
+    \label{fig:PrefPaneSpelling}
 \end{figure}
 
 On the left side, select those dictionaries which should appear in the Mac interface. For each selected dictionary, turn on ``Latex'' and possibly ``Context''. Notice that all dictionaries are variants of English. If you use another language, go to the web page to obtain and install an appropriate dictionary for that language.
@@ -3486,12 +3486,12 @@ On the left side, select those dictionaries which should appear in the Mac inter
 
 Now in the TeXShop Edit menu, show ``Show Spelling and Grammar.''  A panel will appear, see Figure \ref{fig:Spelling_and_Grammar}. At the bottom of this panel, select a cocoAspell dictionary instead of a built-in dictionary. Once it is selected, LaTeX keywords will not be marked as misspelled. Your editor should remember this choice across reboots. 
 
-\section{Coco-Aspell Installing (Schulz)}
+\section{cocoAspell Installing (Schulz)}
 
 
 \subsection{Introduction}
 
-\cmd{CocoAspell} is a \LaTeX-aware Spell Checker that operates through Apple's \cmd{Spell Checking} System. Version 2.5, for \cmd{macOS 10.11} (\cmd{El Capitan}) and later, is available at <\url{http://people.ict.usc.edu/~leuski/cocoaspell/}> or as part of \cmd{MacTeXtras} available at <\url{http://www.tug.org/mactex/mactextras.html}>. 
+\cmd{cocoAspell} is a \LaTeX-aware Spell Checker that operates through Apple's \cmd{Spell Checking} System. Version 2.5, for \cmd{macOS 10.11} (\cmd{El Capitan}) and later, is available at <\url{http://people.ict.usc.edu/~leuski/cocoaspell/}> or as part of \cmd{MacTeXtras} available at <\url{http://www.tug.org/mactex/mactextras.html}>.
 
 If you've never installed \cmd{cocoAspell} before \emph{and} you are using \cmd{OS X 10.13.x} (High Sierra) (or later?) the install may partially fail; the dictionaries are not completely built under that OS version (or later?). There is a fix, provided by Thaddeus Perala, but it involves using the Command Line (\acr{CL}) tool called \cmd{make}.
 
