diff --git a/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib
index a6e6298..7c1130b 100644
--- a/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="21701" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="21701"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="21701"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="21701"/>
diff --git a/Resources/Interfaces/de.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/de.lproj/TSDocument.nib/designable.nib
index 81d7ddb..c210e09 100644
--- a/Resources/Interfaces/de.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/de.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="21701" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="21701"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="21701"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="21701"/>
diff --git a/Resources/Interfaces/es.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/es.lproj/TSDocument.nib/designable.nib
index 6d302f4..9fec65f 100644
--- a/Resources/Interfaces/es.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/es.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/fr.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/fr.lproj/TSDocument.nib/designable.nib
index 1612e84..eee4dd7 100644
--- a/Resources/Interfaces/fr.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/fr.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/it.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/it.lproj/TSDocument.nib/designable.nib
index 74ed22d..b8bf10b 100644
--- a/Resources/Interfaces/it.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/it.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/ja.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/ja.lproj/TSDocument.nib/designable.nib
index b6f82f4..5d42b35 100644
--- a/Resources/Interfaces/ja.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/ja.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/ko.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/ko.lproj/TSDocument.nib/designable.nib
index bd8518f..80c8f68 100644
--- a/Resources/Interfaces/ko.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/ko.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/nl.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/nl.lproj/TSDocument.nib/designable.nib
index 598af5e..152d3c8 100644
--- a/Resources/Interfaces/nl.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/nl.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/pt-PT.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/pt-PT.lproj/TSDocument.nib/designable.nib
index e5caa7c..870d7b0 100644
--- a/Resources/Interfaces/pt-PT.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/pt-PT.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/pt.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/pt.lproj/TSDocument.nib/designable.nib
index db1786e..042b32b 100644
--- a/Resources/Interfaces/pt.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/pt.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="21701" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="21701"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="21701"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="21701"/>
diff --git a/Resources/Interfaces/ro.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/ro.lproj/TSDocument.nib/designable.nib
index 7242019..19e913f 100644
--- a/Resources/Interfaces/ro.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/ro.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="19529" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="19529"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="19529"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="19529"/>
diff --git a/Resources/Interfaces/zh_CN.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/zh_CN.lproj/TSDocument.nib/designable.nib
index 26ac163..b061371 100644
--- a/Resources/Interfaces/zh_CN.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/zh_CN.lproj/TSDocument.nib/designable.nib
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.Cocoa.XIB" version="3.0" toolsVersion="21701" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES">
     <dependencies>
-        <deployment version="120000" identifier="macosx"/>
+        <deployment identifier="macosx"/>
         <plugIn identifier="com.apple.InterfaceBuilder.CocoaPlugin" version="21701"/>
         <plugIn identifier="com.apple.WebKit2IBPlugin" version="21701"/>
         <plugIn identifier="com.apple.pdfkit.ibplugin" version="21701"/>
