diff --git a/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib b/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib
index 7c1130b..c66450d 100644
--- a/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib
+++ b/Resources/Interfaces/Base.lproj/TSDocument.nib/designable.nib
@@ -121,7 +121,7 @@
         <customObject id="-1" userLabel="First Responder" customClass="FirstResponder"/>
         <customObject id="-3" userLabel="Application" customClass="NSObject"/>
         <window title="Preview" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" visibleAtLaunch="NO" frameAutosaveName="PdfWindow" animationBehavior="default" toolbarStyle="expanded" id="26" userLabel="PDF Window" customClass="TSPreviewWindow">
-            <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" resizable="YES" fullSizeContentView="YES"/>
+            <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" resizable="YES"/>
             <windowPositionMask key="initialPositionMask" leftStrut="YES" rightStrut="YES" topStrut="YES" bottomStrut="YES"/>
             <rect key="contentRect" x="798" y="248" width="637" height="709"/>
             <rect key="screenRect" x="0.0" y="0.0" width="2560" height="1415"/>
@@ -2288,7 +2288,7 @@ Gw
             </view>
         </window>
         <window title="Full Window" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" visibleAtLaunch="NO" frameAutosaveName="FullSplitWindow" animationBehavior="default" id="yI2-uy-7pJ" customClass="TSFullSplitWindow">
-            <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" resizable="YES" fullSizeContentView="YES"/>
+            <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" resizable="YES"/>
             <windowCollectionBehavior key="collectionBehavior" fullScreenPrimary="YES"/>
             <windowPositionMask key="initialPositionMask" leftStrut="YES" rightStrut="YES" topStrut="YES" bottomStrut="YES"/>
             <rect key="contentRect" x="283" y="305" width="791" height="336"/>
