diff --git a/Resources/Interfaces/de.lproj/Localizable.strings b/Resources/Interfaces/de.lproj/Localizable.strings
index 177681d..fec4c30 100644
--- a/Resources/Interfaces/de.lproj/Localizable.strings
+++ b/Resources/Interfaces/de.lproj/Localizable.strings
@@ -55,8 +55,8 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "Die Dateikodierung hat sich geändert. Einige Zeichen können bei falscher Kodierung verloren gehen.";
 "Currently open files retain their old encoding." = "Gerade geöffnete Dateien behalten ihre alte Kodierung.";
 "Drag && Drop" = "Drag && Drop";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "Die Verknüpfung /Library/TeX/texbin existiert nicht. Besuchen Sie https://tug.org/mactex und installieren Sie MacTeX oder BasicTeX (2015 oder neuer).";
-"%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ existiert nicht. TeXShop ist ein Frontend für TeX, benötigt aber eine TeX-Distribution. Möglicherweise haben Sie keine Distribution installiert oder diese wurde während eines Betriebssystem-Updates entfernt. Sollte dies der Fall sein, besuchen Sie https://tug.org/mactex und folgen Sie den Installationsanweisungen für MacTeX oder BasicTex. ";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "Die Verknüpfung /Library/TeX/texbin existiert nicht. Besuchen Sie https://tug.org/mactex und installieren Sie MacTeX oder BasicTeX (2015 oder neuer).";
+"%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ existiert nicht. TeXShop ist ein Frontend für TeX, benötigt aber eine TeX-Distribution. Möglicherweise haben Sie keine Distribution installiert oder diese wurde während eines Betriebssystem-Updates entfernt. Sollte dies der Fall sein, besuchen Sie https://tug.org/mactex und folgen Sie den Installationsanweisungen für MacTeX oder BasicTeX.";
 "Can't find required tool." = "Das benötigte Hilfprogramm kann nicht gefunden werden.";
 "Typesetting engine cannot be found." = "Das Schriftsatz-Programm kann nicht gefunden werden.";
 "The source LaTeX document cannot be found." = "Das LaTeX-Quelldokument kann nicht gefunden werden.";
diff --git a/Resources/Interfaces/en.lproj/Localizable.strings b/Resources/Interfaces/en.lproj/Localizable.strings
index afe3a3d..6733090 100644
--- a/Resources/Interfaces/en.lproj/Localizable.strings
+++ b/Resources/Interfaces/en.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "The file encoding changed; characters can be lost if the wrong encoding is used.";
 "Currently open files retain their old encoding." = "Currently open files retain their old encoding.";
 "Drag && Drop" = "Drag && Drop";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX.";
 "Can't find required tool." = "Can't find required tool.";
 "Typesetting engine cannot be found." = "Typesetting engine cannot be found.";
diff --git a/Resources/Interfaces/es.lproj/Localizable.strings b/Resources/Interfaces/es.lproj/Localizable.strings
index 9508e0a..dd284a7 100644
--- a/Resources/Interfaces/es.lproj/Localizable.strings
+++ b/Resources/Interfaces/es.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "La codificación del archivo ha cambiado; se pueden perder caracteres si se usa una codificación equivocada.";
 "Currently open files retain their old encoding." = "Los archivos actualmente abiertos mantienen su codificación antigua.";
 "Drag && Drop" = "Arrastar y soltar";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "El enlace /Library/TeX/texbin no existe. Para solucionarlo, ve a https://tug.org/mactex e instala MacTeX o Basic TeX (2015 o posterior).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "El enlace /Library/TeX/texbin no existe. Para solucionarlo, ve a https://tug.org/mactex e instala MacTeX o BasicTeX (2015 o posterior).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ no existe. TeXShop es una interfaz gráfica para TeX, pero también necesitas una distribución de TeX. Quizás no has instalado ninguna distribución, o se ha borrado durante una actualización del sistema. En ese caso, ve a https://tug.org/mactex y sigue las instrucciones para instalar MacTeX o BasicTeX.";
 "Can't find required tool." = "No puedo encontrar la herramienta requerida.";
 "Typesetting engine cannot be found." = "No puedo encontrar el motor de composición.";
diff --git a/Resources/Interfaces/fr.lproj/Localizable.strings b/Resources/Interfaces/fr.lproj/Localizable.strings
index 1b2ecab..fd713c1 100644
--- a/Resources/Interfaces/fr.lproj/Localizable.strings
+++ b/Resources/Interfaces/fr.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "L'encodage du fichier a changé ; des caractères risquent d'être perdus si le mauvais encodage est utilisé.";
 "Currently open files retain their old encoding." = "Les fichiers déjà ouverts gardent leurs encodages.";
 "Drag && Drop" = "Glisser && Déposer";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "Le lien /Library/TeX/texbin n’existe pas. Pour le réparer, allez à cette adresse https://tug.org/mactex et installez MacTeX ou BasicTeX (2015 ou ultérieur).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "Le lien /Library/TeX/texbin n’existe pas. Pour le réparer, allez à cette adresse https://tug.org/mactex et installez MacTeX ou BasicTeX (2015 ou ultérieur).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ n’existe pas. TeXShop est un outil de gestion pour TeX, mais vous avez aussi besoin d’une distribution TeX. Peut-être qu’une telle distribution n’a pas été installée ou a été supprimée lors d’une mise à niveau de votre système. Si c’est le cas, allez à cette adresse https://tug.org/mactex et suivez les instructions pour installer MacTeX ou BasicTeX.";
 "Can't find required tool." = "N'a pu trouver l'outil requis.";
 "Typesetting engine cannot be found." = "Le moteur TeX n'a pas été trouvé.";
diff --git a/Resources/Interfaces/it.lproj/Localizable.strings b/Resources/Interfaces/it.lproj/Localizable.strings
index 20deedc..cc75ab1 100644
--- a/Resources/Interfaces/it.lproj/Localizable.strings
+++ b/Resources/Interfaces/it.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "La codifica del file è cambiata; alcuni caratteri possono essere persi se si usa una codifica non appropriata.";
 "Currently open files retain their old encoding." = "I file attualmente aperti mantengono la propria codifica.";
 "Drag && Drop" = "Drag && Drop";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "Il link /Library/TeX/texbin non esiste. Per correggere il problema, visita https://tug.org/mactex e installa MacTeX o Basic TeX (2015 o oltre).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "Il link /Library/TeX/texbin non esiste. Per correggere il problema, visita https://tug.org/mactex e installa MacTeX o BasicTeX (2015 o oltre).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ non esiste. TeXShop è un front end per TeX, ma richiede anche l’installazione di una distribuzione TeX. Forse tale distribuzione non è stata installata oppure è stata rimossa durante un aggiornamento di sistema. Se è così, visita https://tug.org/mactex e segui le istruzioni per installare MacTeX o BasicTeX.";
 "Can't find required tool." = "Can't find required tool.";
 "Typesetting engine cannot be found." = "Typesetting engine cannot be found.";
diff --git a/Resources/Interfaces/ja.lproj/Localizable.strings b/Resources/Interfaces/ja.lproj/Localizable.strings
index c7cc557..7331342 100644
--- a/Resources/Interfaces/ja.lproj/Localizable.strings
+++ b/Resources/Interfaces/ja.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "ファイルのエンコーディングが変更されました。違うエンコーディングを使うと、文字情報が失われることがあります。";
 "Currently open files retain their old encoding." = "現在開かれているファイルのエンコーディングは変更されません。";
 "Drag && Drop" = "ドラッグ＆ドロップ"; // zenitani 1.33
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "/Library/TeX/texbin が見つかりません。これを修正するには、https://tug.org/mactex から MacTeX / BasicTeX を再インストールし、TeX Wiki https://texwiki.texjp.org/?Mac の記述を参照して日本語設定を行ってください。";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "/Library/TeX/texbin が見つかりません。これを修正するには、https://tug.org/mactex から MacTeX / BasicTeX を再インストールし、TeX Wiki https://texwiki.texjp.org/?Mac の記述を参照して日本語設定を行ってください。";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ が見つかりません。TeXShop は TeX のフロントエンドに過ぎず、別途 TeX 本体のインストールが必要です。TeX 本体がインストールされていないか、システムのアップグレードの際に削除された可能性があります。もしそうなら、TeX Wiki https://texwiki.texjp.org/?Mac の記述を参照して、https://tug.org/mactex/ から MacTeX / BasicTeX をインストールするなどの対応をとってください。";
 "Can't find required tool." = "必要なツールが見つかりません。";
 "Typesetting engine cannot be found." = "タイプセットエンジンが見つかりません。";
diff --git a/Resources/Interfaces/ko.lproj/Localizable.strings b/Resources/Interfaces/ko.lproj/Localizable.strings
index 03995be..b534044 100644
--- a/Resources/Interfaces/ko.lproj/Localizable.strings
+++ b/Resources/Interfaces/ko.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "The file encoding changed; characters can be lost if the wrong encoding is used.";
 "Currently open files retain their old encoding." = "Currently open files retain their old encoding.";
 "Drag && Drop" = "Drag && Drop";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX.";
 "Can't find required tool." = "Can't find required tool.";
 "Typesetting engine cannot be found." = "Typesetting engine cannot be found.";
diff --git a/Resources/Interfaces/nl.lproj/Localizable.strings b/Resources/Interfaces/nl.lproj/Localizable.strings
index 66b3101..74ab534 100644
--- a/Resources/Interfaces/nl.lproj/Localizable.strings
+++ b/Resources/Interfaces/nl.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "The file encoding changed; characters can be lost if the wrong encoding is used.";
 "Currently open files retain their old encoding." = "Currently open files retain their old encoding.";
 "Drag && Drop" = "Drag && Drop";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX.";
 "Can't find required tool." = "Can't find required tool.";
 "Typesetting engine cannot be found." = "Typesetting engine cannot be found.";
diff --git a/Resources/Interfaces/pt-PT.lproj/Localizable.strings b/Resources/Interfaces/pt-PT.lproj/Localizable.strings
index d178f7b..ac6da6b 100644
--- a/Resources/Interfaces/pt-PT.lproj/Localizable.strings
+++ b/Resources/Interfaces/pt-PT.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "The file encoding changed; characters can be lost if the wrong encoding is used.";
 "Currently open files retain their old encoding." = "Currently open files retain their old encoding.";
 "Drag && Drop" = "Drag && Drop";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX.";
 "Can't find required tool." = "Can't find required tool.";
 "Typesetting engine cannot be found." = "Typesetting engine cannot be found.";
diff --git a/Resources/Interfaces/pt.lproj/Localizable.strings b/Resources/Interfaces/pt.lproj/Localizable.strings
index 03d2241..5d3e392 100644
--- a/Resources/Interfaces/pt.lproj/Localizable.strings
+++ b/Resources/Interfaces/pt.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "A codificação do arquivo foi alterada; caracteres podem ser perdidos se uma codificação errada for usada.";
 "Currently open files retain their old encoding." = "Arquivos abertos continuarão a manter a codificação antiga.";
 "Drag && Drop" = "Arrastar && Soltar";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "O link /Library/TeX/texbin não existe. Para corrigir isso vá no site https://tug.org/mactex e instale o MacTeX ou BasicTeX (2015 ou posterior).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "O link /Library/TeX/texbin não existe. Para corrigir isso vá no site https://tug.org/mactex e instale o MacTeX ou BasicTeX (2015 ou posterior).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ não existe. TeXShop é um aplicativo gráfico para TeX, sendo assim você precisa de uma distribuição TeX. Possivelmente essa distribuição não está instalada ou foi removida durante uma atualização do sistema. Se for isso, então vá no site https://tug.org/mactex e siga as instruções para instalar o MacTeX ou BasicTeX.";
 "Can't find required tool." = "Não foi possível encontrar a ferramenta necessária.";
 "Typesetting engine cannot be found." = "Motor de compilação não pode ser encontrado.";
diff --git a/Resources/Interfaces/ro.lproj/Localizable.strings b/Resources/Interfaces/ro.lproj/Localizable.strings
index 2876381..ee4b1d6 100644
--- a/Resources/Interfaces/ro.lproj/Localizable.strings
+++ b/Resources/Interfaces/ro.lproj/Localizable.strings
@@ -55,7 +55,7 @@
 "The file encoding changed; characters can be lost if the wrong encoding is used." = "The file encoding changed; characters can be lost if the wrong encoding is used.";
 "Currently open files retain their old encoding." = "Currently open files retain their old encoding.";
 "Drag && Drop" = "Drag && Drop";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later).";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later).";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX.";
 "Can't find required tool." = "Can't find required tool.";
 "Typesetting engine cannot be found." = "Typesetting engine cannot be found.";
diff --git a/Resources/Interfaces/zh_CN.lproj/Localizable.strings b/Resources/Interfaces/zh_CN.lproj/Localizable.strings
index 70f74f0..8cc51f7 100644
--- a/Resources/Interfaces/zh_CN.lproj/Localizable.strings
+++ b/Resources/Interfaces/zh_CN.lproj/Localizable.strings
@@ -1,8 +1,8 @@
         "%@\n\nCharacters: %d" = "%@\n\nCharacters: %d";
     "%@ does not exist." = "%@ \U4e0d\U5b58\U5728\U3002";
-"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or Basic TeX (2015 or later)." = "目录链接 /Library/TeX/texbin 不存在。若要修复，请访问 https://tug.org/mactex 并安装 MacTeX 或 Basic TeX (2015 或更新版)。";
+"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)." = "目录链接 /Library/TeX/texbin 不存在。若要修复，请访问 https://tug.org/mactex 并安装 MacTeX 或 BasicTeX (2015 或更新版)。";
 "%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX." = "%@ 不存在。TeXShop 仅为 TeX 的前端，您还需要安装 TeX 发行包。发行包可能尚未安装，或在系统升级时被删除。若有此 况，请访问 https://tug.org/mactex 并按照说明安装 MacTeX 或 BasicTeX。";
-"%@ does not have the executable bit set." = "%@ 不存在。TeXShop 仅为 TeX 的前端，您还需要安装 TeX 发行包。发行包可能尚未安装，或在系统升级时被删除。若有此 况，请访问 http://tug.org/mactex 并按照说明安装 MacTeX 或 BasicTeX。";
+"%@ does not have the executable bit set." = "%@ 不存在。TeXShop 仅为 TeX 的前端，您还需要安装 TeX 发行包。发行包可能尚未安装，或在系统升级时被删除。若有此 况，请访问 https://tug.org/mactex 并按照说明安装 MacTeX 或 BasicTeX。";
     "Add macros from file..." = "\U4ece\U6587\U4ef6\U6dfb\U52a0\U5b8f...";
     AutoComplete = "\U81ea\U52a8\U66f4\U6b63";
     Autocompletion = "\U81ea\U52a8\U66f4\U6b63";
diff --git a/Sources/TSDocument-Jobs.m b/Sources/TSDocument-Jobs.m
index 54153dd..b385efd 100644
--- a/Sources/TSDocument-Jobs.m
+++ b/Sources/TSDocument-Jobs.m
@@ -1105,8 +1105,8 @@ - (BOOL) startTask: (NSTask*) task running: (NSString*) leafname withArgs: (NSMu
             {
             NSBeginAlertSheet(NSLocalizedString(@"Can't find required tool.", @"Can't find required tool."),
                           nil, nil, nil, [textView window], nil, nil, nil, nil,
-                          NSLocalizedString(@"The link /Library/TeX/texbin does not exist. To fix this, go to http://tug.org/mactex and install MacTeX or Basic TeX (2015 or later). Or for a much smaller download, go to http://pages.uoregon.edu/koch and install FixLink.",
-                            @"The link /Library/TeX/texbin does not exist. To fix this, go to http://tug.org/mactex and install MacTeX or Basic TeX (2015 or later). Or for a much smaller download, go to http://pages.uoregon.edu/koch and install FixLink."),
+                          NSLocalizedString(@"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later).",
+                            @"The link /Library/TeX/texbin does not exist. To fix this, go to https://tug.org/mactex and install MacTeX or BasicTeX (2015 or later)."),
                           filename);
                 return FALSE;
             }
@@ -1114,8 +1114,8 @@ - (BOOL) startTask: (NSTask*) task running: (NSString*) leafname withArgs: (NSMu
             {
             NSBeginAlertSheet(NSLocalizedString(@"Can't find required tool.", @"Can't find required tool."),
 						  nil, nil, nil, [textView window], nil, nil, nil, nil,
-						  NSLocalizedString(@"%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to http://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX.",
-                            @"%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to http://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX."),
+						  NSLocalizedString(@"%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX.",
+                            @"%@ does not exist. TeXShop is a front end for TeX, but you also need a TeX distribution. Perhaps such a distribution was not installed or was removed during a system upgrade. If so, go to https://tug.org/mactex and follow the instructions to install MacTeX or BasicTeX."),
 						  filename);
                 return FALSE;
             }
