diff --git a/Sources/TSToolbarController.m b/Sources/TSToolbarController.m
index 533a5e2..dfc5661 100644
--- a/Sources/TSToolbarController.m
+++ b/Sources/TSToolbarController.m
@@ -642,15 +642,7 @@ - (NSToolbarItem *) toolbar: (NSToolbar *)toolbar itemForItemIdentifier: (NSStri
         return toolbarItem;
     }
     //NDS added dropdown for going to a label
-
-    
     if ([itemIdent isEqual: skLabelsTID]) {
-        
-        BOOL NotGerman = YES;
-        NSString *testString = NSLocalizedString(@"Cancel", @"Cancel");
-        if ([testString isEqualToString: @"Abbrechen"])
-            NotGerman = NO;
-        
         toolbarItem = [self makeToolbarItemWithItemIdentifier:itemIdent key:itemIdent customView:slabels];
          menuFormRep = [[NSMenuItem alloc] init] ;
         
@@ -658,12 +650,9 @@ - (NSToolbarItem *) toolbar: (NSToolbar *)toolbar itemForItemIdentifier: (NSStri
         [menuFormRep setTitle: [toolbarItem label]];
         [toolbarItem setMenuFormRepresentation: menuFormRep];
         theSLabels = toolbarItem;
-        if (NotGerman)
-            return toolbarItem;
+        return toolbarItem;
     }
     
-   
-    
     if ([itemIdent isEqual: kUpdateTID]) {
     
         return [self makeToolbarItemWithItemIdentifier:itemIdent key:itemIdent
